### DISCUSSION TITLE
Where is the .env file

### DISCUSSION THREAD

**Discussion by lordratner** [exadel-inc/CompreFace#928] [Q&A]
Hi,

For the life of me I cannot find the .env file. Compreface is installed on a fresh debian LXC. I ran the script in the /opt folder.

Thanks
Seth

**Comment by pospielov** [comment]
It should be in the same folder as docker-compose.yml file.
When you download CompreFace, there is an archive with both docker-compose.yml and  .env files

**Comment by pvi25** [comment]
maybe you have to enable 'show hidden files'? ls -ltr doesnt show it, ls -latr does
